File: microbit-distance/microbit-distance.js
radio.onReceivedNumber(function on_received_number(receivedNumber: number) {
    
    signal = radio.receivedPacket(RadioPacketProperty.SignalStrength)
    basic.showNumber(signal)
})
let signal = 0
radio.setTransmitPower(7)
radio.setGroup(1)
basic.forever(function on_forever() {
    radio.sendNumber(0)
})

File: microbit-distance/microbit-distance.py
def on_received_number(receivedNumber):
    global signal
    signal = radio.received_packet(RadioPacketProperty.SIGNAL_STRENGTH)
    basic.show_number(signal)
radio.on_received_number(on_received_number)

signal = 0
radio.set_transmit_power(7)
radio.set_group(1)

def on_forever():
    radio.send_number(0)
basic.forever(on_forever)
